frontend/components/transaction/SwapTransaction.vue

```diff
@@ -29,27 +29,39 @@
             </Explain>
             the auction. In case your transaction will be rejected, it only results in the loss of the transaction fee.
         </TextBlock>
-        <WalletBlock
-            class="mb-6 lg:mb-0"
-            :disabled="!auctionTransaction.isActive"
-            :is-loading="isConnecting"
+        <WalletConnectionCheckPanel
             :wallet-address="walletAddress"
+            :is-loading="isConnecting"
             :is-explanations-shown="isExplanationsShown"
+            :disabled="!auctionTransaction.isActive"
             @connectWallet="$emit('connect')"
             @disconnectWallet="$emit('disconnect')"
         />
```

Comment:
The idea of the panels is that they also incapsulate reusable `isCorrect` checks. 
See wallet popup for the example how it's used:
https://github.com/sidestream-tech/unified-auctions-ui/blob/a4fbe882660e45f65ba46b1244874abfbceb12a6/frontend/components/wallet/WalletDepositWithdrawBlock.vue#L27

Therefore, please use `isCorrect.sync` props on each panel